presto/presto.py

```diff
@@ -874,11 +868,8 @@ def load_pretrained(
         if dekadal:
             model = extend_to_dekadal(model)
 
-        # if is_finetuned:
-        # here, I want to be able to upload Presto model that
-        # has already been finetuned so that I can only play with the head
-        # a model needs to be constructed so that weights can be loaded
-        # currently, cannot correctly construct the finetuned head and populate it with weights 😥
+        if is_finetuned:
+            model = model.construct_finetuning_model(num_outputs)
```

Comment:
@cbutsko  regarding what was previously here - a comment about something that couldn't be done - what was changed now so it works?

Reply:
nothing really changed, before that I was trying to load finetuned model's state_dict onto a regular presto conctruction 😔 which of course did not work. now I plugged the correct functionality of constructing a finetuned model - and it all works. so it's rather a bug than an issue